tikv-configuration-file.md

```diff
@@ -1300,11 +1300,6 @@ Configuration items related to `rocksdb.defaultcf.titan`.
 + Determines whether to optimize the read performance. When `level-merge` is enabled, there is more write amplification.
 + Default value: `false`
 
-### `gc-merge-rewrite`
```

Comment:
In which version is this config deprecated?

Reply:
Master.

Reply:
I guess it will be deprecated in v6.2, right? Also, deprecation is not the same as deletion. Deprecation means that you are not recommended to but can still use it. Deletion is performed on the code side, then on the doc side.

Reply:
User can still configure it. But they cannot start TiKV with it enabled.